worker/worker.py

```diff
@@ -217,6 +217,12 @@ def setup_parameters(config_file):
     elif protocol == "https" and options.port == "80":
         # Rewrite old port 80 to 443
         options.port = "443"
+    # If we are not going to use all cores, then unset that option.
+    # Note that if we used that option previously, we'd have set
+    # concurrency to the maximum value in the config file, so this will
+    # (only) apply when overruling concurrency via the command line.
+    if cpu_count < multiprocessing.cpu_count():
```

Comment:
This is always `True` if we don't have already `concurrency = <Max Threads>` in the config file, so we are not able anymore to use all Threads with `--use_all_cores True`

Reply:
You're totally right. We probably can't get around to changing options handling so it can make the distinction between command line options and config file ones

Reply:
Yes. `--concurrency` and `--use_all_cores` are a unit. If one of them is present on the command line then neither of them should default to the value in the configfile.